### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by archyposada**
Here are the changes we discussed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,7 +211,7 @@ Also known as the `client_id`. You must additionally pass in your Adobe API key
 
 ## Rate Limiting
 
-We have not put a throttle limit on requests to the API at this time.
+All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information.
```
Comment (khound) [inline README.md:214]: @archyposada Throttling(Rate limiting) and Trial limit are two different things. Let's keep the previous line . May be something like this ? `We have not put a throttle limit on requests to the API at this time.
However we limit Free Trials. All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information`

**Comment by khound** [comment]
@archyposada I have one small comment, otherwise it looks good.

File: README.md (hunk 1 above)
Comment (archyposada) [reply README.md:214]: Why don't we leave the rate limiting section the same with the old sentence and add new section labeled Quota Limits and add:
"All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information. "

Comment (khound) [reply README.md:214]: @archyposada That sounds good to me.

**Pull request merged**